app/frontend/src/pages/chat/Chat.tsx

```diff
@@ -134,6 +134,9 @@ const Chat = () => {
         setActiveCitation(undefined);
         setActiveAnalysisPanelTab(undefined);
         setAnswers([]);
+        setstreamedAnswers([]);
```

Comment:
Sorry I didn't notice in last PR, can you rename setstreamedAnswers to setStreamedAnswers, here and above? (Proper camel-case). This looks reasonable otherwise, all of these seem like attributes that get set on the next question when needed.

Reply:
Done